lib/ansible/module_utils/powershell/Ansible.ModuleUtils.Backup.psm1

```diff
@@ -0,0 +1,24 @@
+# Copyright (c): 2018, Dag Wieers (@dagwieers) <dag@wieers.com>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+Set-StrictMode -Version 2.0
+$ErrorActionPreference = "Stop"
+
+Function Backup-File($path, $obj=@{}, $check_mode=$false) {
+<#
+    .SYNOPSIS
+    Helper function to make a backup of a file.
+    .EXAMPLE
+    Backup-File -path $path -obj $result
+#>
+    $backup_path = "$path.$pid." + [DateTime]::Now.ToString("yyyyMMdd-HHmmss") + ".bak";
+    Try {
+        Copy-Item -Path $path -Destination $backup_path -WhatIf:$check_mode
```

Comment:
I would use `-LiteralPath` instead of `-Path`. We have no need for glob like operations. If you think we should support it then potentially adding `$Path` or `$LiteralPath` to this cmdlet may be the better option but I really don't think we should support that.

Reply:
Good catch, there's still a topic open to whether we should evaluate the use of `-Path` in other modules. Not sure if that is still relevant.

Reply:
Glob-like operations makes no sense here.